catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/cql/CqlRequest.java

```diff
@@ -203,6 +223,22 @@ private String parseSrc(String localSource) {
     return src;
   }
 
+  private void parseSrcs(String localSource) {
+    for (int i = 0; i < srcs.length; i++) {
+      if (StringUtils.equalsIgnoreCase(srcs[i], LOCAL_SOURCE) || StringUtils.isBlank(srcs[i])) {
```

Comment:
in what situations will `srcs` have blank strings?

Reply:
I extrapolated this function from the `parseSrc` function that was already present for a single source and this is the check that was used there. I suppose there wouldn't in practice be a case where `srcs`has a blank string - I will remove the `isBlank` portion